Contents/mods/More Traits/media/lua/client/MoreTraits.lua

```diff
@@ -871,11 +873,21 @@ local function indefatigablecounter()
     end
 end
 
-local function badteethtrait(_player)
+local function badteethtrait(_player, _playerdata)
     local player = _player;
+    local playerdata = _playerdata;
+    local healthtimer = player:getBodyDamage():getHealthFromFoodTimer();
     if player:HasTrait("badteeth") then
-        if player:getBodyDamage():getHealthFromFoodTimer() > 1000 then
-            player:getStats():setPain(player:getBodyDamage():getHealthFromFoodTimer() / 90);
+        if healthtimer > 1000 then
+            if healthtimer > playerdata.fPreviousHealthFromFoodTimer then
+                local Head = player:getBodyDamage():getBodyPart(BodyPartType.FromString("Head"));
+                local pain = player:getBodyDamage():getHealthFromFoodTimer() * 0.01;
+                Head:setAdditionalPain(Head:getAdditionalPain() + pain);
+                print("pain: " .. pain)
+                print("healthtimer: " .. healthtimer);
+                print("previous ht: " .. playerdata.fPreviousHealthFromFoodTimer);
```

Comment:
I would suggest removing these print commands to avoid log spam.

Reply:
These print commands trigger once per consumed food, if this food also causes players teeth to hurt.